### PR TITLE
Remove oauth from iv/tv

### PR DESCRIPTION
**Pull request by trav3711**
Removing Oauth as the primary authentication method for telemetry validation. Replacing with gcloud cli application default credentials.

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -107,7 +116,8 @@ python toolkit.py -i //path/to/building/configuration/file.yaml -g -v -s subscri
 
 results in these actions:
 1. Ingests a building configuration file.
-2. Generates guids for every entity instance in the buiding configuration file and re-writes building configuration file in the new format.
+2. Generates guids for every entity instance in the building configuration 
+   file and re-writes building configuration file in the new format.
```
Comment (cstirdivant) [inline tools/README.md:120]: nit: remove weird carriage return?

File: tools/README.md [hunk 2]
```diff
@@ -87,12 +95,13 @@ Run `python toolkit.py` and provide the following arguments:
 
 4. `--validate` or `-v`: Runs instance validator to validate the building configuration file.
 
-5. [Optional] Telemetry Validation: After a building configuration's entity types are validated, validation must also be run on the telemetry payload using the following parameters. **NOTE:** The OAuth credential (`-c`) and subscription (`-s`) are provided by the Google team. Please reach out to your IoT TPM for guidance.
+5. [Optional] Telemetry Validation: After a building configuration's entity
```
Comment (cstirdivant) [inline tools/README.md:98]: nit: remove carriage returns

**Comment by trav3711** [comment]
The carriage returns are automatic for Pycharm. They won't affect the actual READMEs. Please ignore.

**Comment by cstirdivant** [comment]
LGTM

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**